cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSICKANSink.java

```diff
@@ -459,6 +512,11 @@ public void aggregate(NGSIEvent event) {
             NotifyContextRequest.ContextElement contextElement = event.getContextElement();
             String entityId = contextElement.getId();
             String entityType = contextElement.getType();
+            //LOGGER.debug("$$$$$$$$$$$$ "+entityId+"$$$$$$$$$$$$$");
```

Comment:
leftover code?

Reply:
Fixed in f1f9737cf4ef5d7a113be681a47790b8d860c0f3